mod-login-saml/src/main/resources/domain/mod-login-saml/features/loginSamlPost.feature

```diff
@@ -0,0 +1,48 @@
+Feature: Login SAML with a POST binding
+
+  Background:
+    * url baseUrl
+    * callonce login testUser
+    * configure headers =
+    """
+    {
+      "X-Okapi-Tenant": "#(testTenant)",
+      "X-Okapi-Token": "#(okapitoken)",
+      "Accept": "application/json, text/plain"
+    }
+    """
+
+  Scenario: Do preflight request, POST to saml login endpoint, and receive correct response for POST binding
+    Given path "saml/login"
```

Comment:
The actual OPTIONS request uses this uri: https://folio-snapshot-okapi.dev.folio.org/_/invoke/tenant/diku/saml/login

However, we should keep the test for the wrong path "saml/login" and check that the credentials header is NOT present:
```
  Background:
    * configure lowerCaseResponseHeaders = true
```

`And match responseHeaders contains { 'access-control-allow-credentials': '#notpresent' }`

And add a test for the correct path `"_/invoke/tenant/" + testTenant + "/saml/login"`

Reply:
Indeed you are right,  ```saml/login``` also goes through ```_/invoke/tenant/...```. I'm also noticing that the current test looks for the response header to be ```header access-control-allow-origin: "*"``` when I think it should be ```header access-control-allow-origin: <the origin url>```.

Noticing too that the ```saml/configure``` and perhaps other tests could have OPTIONS requests on them too. Will add those.

I also like the use of ```lowerCaseResponseHeaders``` and ```#notpresent``` on ```access-control-allow-credentials``` for a new test that checks the wrong path.